draft-ietf-rats-reference-interaction-models.md

```diff
@@ -326,6 +326,74 @@ As soon as the Verifier receives the Evidence and the Event Logs, it appraises t
+two step procedure. In the first step, an Attester conveys Evidence to a Verifier which compares the Evidence against its appraisal policy.  The Verifier
+then gives back an Attestation Result to the Attester, which simply caches it. In the second step, the Attester presents the Attestation Result (and possibly additional Claims/evidence) to a Relying Party, which then compares this information against its own appraisal policy to establish the trustworthiness of the attestor.
+
+~~~~
+.----------.                                                .----------.                            .----------.
+| Attester |                                                | Verifier |                            | R. P.    |
+'----------'                                                '----------'                            '----------'
+     |                                                            |                                      |
+  generateClaims(attestingEnvironment)                            |                                      |
+     | => claims, eventLogs                                       |                                      |
+     |                                                            |                                      |
+     | <-- requestAttestation(handle, authSecIDs, claimSelection) |                                      |
+     |                                                            |                                      |
+  collectClaims(claims, claimSelection)                           |                                      |
+     | => collectedClaims                                         |                                      |
+     |                                                            |                                      |
+  generateEvidence(handle, authSecIDs, collectedClaims)           |                                      |
+     | => evidence                                                |                                      |
+     |                                                            |                                      |
+     | evidence, eventLogs -------------------------------------> |                                      |
+     |                                                            |                                      |
+     |                appraiseEvidence(evidence, eventLogs, refValues)                                   |
+     |                                                            |                                      |
+     |   attestationResults <-----------------------------------  |                                      |
+     |                                                            |                                      |
+     | attestationResults(evidence, results) ----------------------------------------------------------> |      |                                                            |                                      |
```

Comment:
there seems to be a problem with NL here

Reply:
Sure thanks Thomas, will look into this later today!